.github/workflows/bump_plugins.yml

```diff
@@ -0,0 +1,107 @@
+name: Update plugins
+on:
+  workflow_dispatch:
+
+jobs:
+  # ugly way to share the plugin list
+  plugin_list:
+    runs-on: ubuntu-latest
+    outputs:
+      plugins: ${{ steps.set-list.outputs.plugins }}
+    steps:
+    - name: Write the plugin list
+      run: echo '["foreman_ansible"]' > plugin_list.json
+    - name: Set the plugins list
+      id: set-list
+      run: |
+        PLUGINS=$(echo $(cat plugin_list.json))
+        echo "::set-output name=plugins::$PLUGINS"
+
+  # Bump the plugins on RPM
+  bump_plugin_rpm:
+    needs: plugin_list
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        plugin:
+          - ${{ fromJson(needs.plugin_list.outputs.plugins) }}
+
+    env:
+      PLUGIN_DIR: "packages/plugins/rubygem-${{ matrix.plugin }}"
```

Comment:
This is only used once. Perhaps just inline it?

On the other hand, I can also imagine we end up with a script that collects directories to update and make that the test matrix. That seems likely given that the list of plugins depends on what we package for.

Reply:
I've inlined it, if we use lets refactor once we would actually use it twice.